opstack/src/consensus.rs

```diff
@@ -33,13 +41,15 @@ impl ConsensusClient {
 
         let mut inner = Inner {
             server_url: config.consensus_rpc.to_string(),
-            unsafe_signer: config.chain.unsafe_signer,
+            unsafe_signer: Arc::new(Mutex::new(config.chain.unsafe_signer)),
             chain_id: config.chain.chain_id,
             latest_block: None,
             block_send,
             finalized_block_send,
         };
 
+        verify_unsafe_signer(inner.server_url.clone(), inner.unsafe_signer.clone());
```

Comment:
Not awaiting this future means it won't necessarily run. We should use the `run` function  to ensure it happens in the background.

Reply:
We use the run func inside it here https://github.com/a16z/helios/blob/7677f593b7d396c7b2e9b90b9dba58afedba6ef8/opstack/src/consensus.rs#L162